numpy/doc/creation.py

```diff
@@ -9,8 +9,7 @@
 There are 5 general mechanisms for creating arrays:
 
 1) Conversion from other Python structures (e.g., lists, tuples)
-2) Intrinsic numpy array array creation objects (e.g., arange, ones, zeros,
-   etc.)
+2) Intrinsic numpy array creation objects (e.g., arange, ones, zeros, etc.)
```

Comment:
now the line is 81 chars, which is over the limit (80)

Reply:
Line length didn't change here